packages/preferences/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### Breaking change
+
+-   The preferences package is no longer compatible with the `@wordpress/data` persistence plugin. Please use the new `setPersistenceLayer` API. ([#39795](https://github.com/WordPress/gutenberg/pull/39795))
```

Comment:
There was one, I removed it in this commit - https://github.com/WordPress/gutenberg/pull/39795/commits/c171bb53b061b5dd07b95c71c1d51176b880a9e4

The main reason being that shipping unused code is something the project tends to avoid. It's also not straightforward to have a default, as the data is fetched asynchronously in an action. I did a lot of headscratching, but couldn't figure out how to make that work.